templates/ansible/resource.erb

```diff
@@ -257,12 +266,54 @@ def main():
   )
 -%>
     return <%= method %>
+<% else # non-updatable, but has per-field updates -%>
+<%
+  method = method_call('fetch_resource', ['module', 'self_link(module)',
+                                          ('kind' if object.kind?),
+                                         ])
+-%>
+    return <%= method %>
 <%   end # if object.input -%>
 <% else # object.update.nil? -%>
 <%= lines(indent(object.update, 4)) -%>
 <% end # object.update.nil? -%>
 
 
+<% unless update_props.empty? -%>
+def update_fields(module, request, response):
+<% empty = update_props.values.flatten.reject { |p| p.is_a? Api::Type::FetchedExternal }.empty? -%>
```

Comment:
There's some rules in GOVERNANCE.MD about how to indent this stuff ... I'm not too bothered, honestly, but wanted to let you know.  Dana recently changed `terraform/resource.erb` to follow those rules and it looks pretty good.